packages/amplify-e2e-core/src/utils/selectors.ts

```diff
@@ -6,7 +6,11 @@ export const moveDown = (chain: ExecutionContext, nMoves: number) =>
 export const moveUp = (chain: ExecutionContext, nMoves: number) =>
   Array.from(Array(nMoves).keys()).reduce((chain) => chain.send('k'), chain);
 
-export const singleSelect = <T>(chain: ExecutionContext, item: T, allChoices: T[]) => multiSelect(chain, [item], allChoices);
+export const singleSelect = <T>(chain: ExecutionContext, item: T, allChoices: T[]) => {
+  moveDown(chain, allChoices.indexOf(item));
```

Comment:
looks good to me if e2e pass, this might break the tests where the default preselected item is not the first in list

Reply:
afaik multiselect suffers from same problem which is solved by things like that https://github.com/aws-amplify/amplify-cli/blob/99026e1a0282bd621eb579fdca8ec3653e8daff2/packages/amplify-e2e-core/src/categories/lambda-function.ts#L340-L341 .